store/postgres/src/dynds.rs

```diff
@@ -1,100 +1,64 @@
 //! SQL queries to load dynamic data sources
 
-use std::ops::Bound;
-
-use diesel::pg::PgConnection;
-use diesel::prelude::{ExpressionMethods, JoinOnDsl, QueryDsl, RunQueryDsl};
+use diesel::{
+    delete,
+    dsl::sql,
+    prelude::{ExpressionMethods, QueryDsl, RunQueryDsl},
+    sql_query,
+    sql_types::Text,
+};
+use diesel::{insert_into, pg::PgConnection};
 
 use graph::{
     components::store::StoredDynamicDataSource,
     constraint_violation,
     data::subgraph::Source,
-    prelude::{bigdecimal::ToPrimitive, web3::types::H160, BigDecimal, StoreError},
+    prelude::{
+        bigdecimal::ToPrimitive, web3::types::H160, BigDecimal, BlockNumber, EthereumBlockPointer,
+        StoreError, SubgraphDeploymentId,
+    },
 };
 
-use crate::block_range::first_block_in_range;
-
-// Diesel tables for some of the metadata
-// See also: ed42d219c6704a4aab57ce1ea66698e7
-// Changes to the GraphQL schema might require changes to these tables.
-// The definitions of the tables can be generated with
-//    cargo run -p graph-store-postgres --example layout -- \
-//      -g diesel store/postgres/src/subgraphs.graphql subgraphs
-// BEGIN GENERATED CODE
 table! {
     subgraphs.dynamic_ethereum_contract_data_source (vid) {
         vid -> BigInt,
-        id -> Text,
-        kind -> Text,
         name -> Text,
-        network -> Nullable<Text>,
-        source -> Text,
-        mapping -> Text,
+        address -> Binary,
+        abi -> Text,
+        start_block -> Integer,
+        // Never read
         ethereum_block_hash -> Binary,
         ethereum_block_number -> Numeric,
         deployment -> Text,
         context -> Nullable<Text>,
-        block_range -> Range<Integer>,
```

Comment:
I think it's easy enough to reintroduce `block_range` when we actually need it. I am trying to cut down on the amount of data we store even though we don't need it. I was going to also delete `ethereum_block_hash` but left it in so that we have an actual block pointer and not just a block number.

Reply:
Fair enough.